services/ui-src/src/components/tables/getEntityStatus.tsx

```diff
@@ -84,12 +84,16 @@ export const getEntityStatus = (
     });
 
     //check to see if each validation id was matched to user selected values
-    return isFilled?.every((field: AnyObject) => {
-      return field && field.length > 0;
-    });
+    if (
+      isFilled?.every((field: AnyObject) => {
+        return field && field.length > 0;
```

Comment:
`field?.length > 0` ?

Reply:
I don't think we need the optional chain if it's combined with `field &&` because the first part ensures it exists

Reply:
Hmm maybe not 🤔 

<img width="336" alt="Screenshot 2024-06-25 at 12 19 02 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/57802560/a7dc31d0-6536-4808-8d7b-ff2f3b91cf0b">

Reply:
Looks like `var name = undefined` actually assigns a string `undefined` so that's not correct behavior.

Reply:
Also `?` for `isFilled?` seems unnecessary because `isFilled` is always an array.

Reply:
hmm that's a good point. I'm gonna leave that alone for now because I saw a lot more opportunities for refactoring here